src/GamecubeAPI.hpp

```diff
@@ -24,6 +24,8 @@ THE SOFTWARE.
 // Include guard
 #pragma once
 
+#include "GamecubeAPI.h"
+
```

Comment:
Why did you add this include here?

Reply:
I was getting compilation errors without it when bundling with my project. I think it is generally best practice anyway to always include the header where its definitions are needed. Seeing as you have include guards/"pragma once" to prevent double inclusion, there's no reason not to, and this will ensure that definitions are always available by the time they are needed.

Reply:
I think this is really not required, as it is included here:
https://github.com/NicoHood/Nintendo/blob/master/src/GamecubeAPI.h#L78

Can you please give it another test, I am very sure this is not required.

Reply:
Hmm okay it does seem to be fine now. Maybe my includes in my project were wrong at the time. I can remove it if you want, although it does also give errors in my editor when the include isn't present.

Reply:
Yes, please remove it. It would be wrong to have the include there, since this is a header implementation, not a c file.

Reply:
Okay, done